### PR TITLE
Detect devices: Odys, Advan, Evercoss, Dapeng, Figo

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -2380,6 +2410,14 @@ Datsun:
     - regex: 'DATSUN_D5500'
       model: 'D5500'
 
+# Dapeng
+Dapeng:
+  regex: 'A75 Build'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: Not sure if that detection doesn't match to many things. Actually there are other devices named A75, like https://www.oppo.com/en/smartphone-a75/ or one from Micromax. Guess we shouldn't add it if it isn't unambiguous?

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -7141,12 +7232,22 @@ Roku:
...
+  regex: 'IO Pro'
+  device: 'smartphone'
+  models:
+    - regex: 'IO Pro'
+      model: 'IO Pro'
+
 # Rover
 Rover:
-  regex: 'Rover ([0-9]+)'
+  regex: 'rokit io pro 3d'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: this change does not seem to make any sense, guess you wanted to add that in the detection before...

**Review by sgiehl — CHANGES_REQUESTED** [review]

File: regexes/device/mobiles.yml (hunk 1 above)
Comment (sanchezzzhak) [reply regexes/device/mobiles.yml]: Remove.

File: regexes/device/mobiles.yml (hunk 2 above)
Comment (sanchezzzhak) [reply regexes/device/mobiles.yml]: Revert change

**Pull request merged**